src/app/external-cluster-wizard/steps/external-cluster/provider/gke/template.html

```diff
@@ -72,35 +107,55 @@
         Kubernetes Version is <strong>required</strong>.
       </mat-error>
     </mat-form-field>
+  </div>
+
+  <div [fxFlex]="isDialogView() ? '100%' : '50%'"
+       fxLayout="column"
+       fxLayoutGap="10px">
+    <ng-container *ngIf="!isDialogView()">
+      <mat-card-header class="km-no-padding">
+        <mat-card-title>Initial Node Pool</mat-card-title>
+      </mat-card-header>
+
+      <mat-form-field>
+        <mat-label>Node Pool Name</mat-label>
+        <input matInput
+               type="text"
+               [formControlName]="Controls.InitialNodePoolName" />
+        <mat-hint>Initial Node Pool Name can't be changed</mat-hint>
+      </mat-form-field>
+    </ng-container>
 
-    <km-autocomplete *ngIf="isDialogView()"
-                     label="Machine Type"
-                     [options]="machineTypes"
-                     [formControlName]="Controls.MachineTypes">
-      <mg-container hint>
-        Select vCPU cores and memory
-      </mg-container>
-    </km-autocomplete>
-
-    <km-autocomplete *ngIf="isDialogView()"
-                     label="Disk Type"
-                     [options]="diskTypes"
-                     [formControlName]="Controls.DiskTypes">
-      <mg-container hint>
-        SSD persistent disk is better for random IOPS or for streaming throughput with low latency.
-      </mg-container>
-    </km-autocomplete>
-
-    <km-number-stepper *ngIf="isDialogView()"
-                       hint="Boot disk size is permanent."
+    <km-combobox filterBy="name"
+                 inputLabel="Select Machine Type..."
+                 hint="Select vCPU cores and memory"
+                 [label]="machineTypeLabel"
+                 [options]="machineTypes"
+                 [formControlName]="Controls.MachineTypes">
+      <div *option="let machineType">
+        {{machineType.name+" ("+machineType.vcpus +" vCPUs "+machineType.memory+"MB RAM)"}}
+      </div>
+    </km-combobox>
+
+    <km-combobox filterBy="name"
+                 inputLabel="Select Disk Type..."
+                 hint="SSD persistent disk is better for random IOPS or for streaming throughput with low latency."
+                 [label]="diskTypeLabel"
+                 [options]="diskTypes"
+                 [formControlName]="Controls.DiskTypes">
+      <div *option="let diskType">
+        {{diskType.description}}
```

Comment:
**Changes should be tested on both (Wizard/MD dialog)**

![disk-type (display-name)](https://user-images.githubusercontent.com/17727069/187876081-0d7360e2-841e-496c-ace0-a6a136b82ce2.png)

**show as**
![showas](https://user-images.githubusercontent.com/17727069/187876313-1db421ff-363d-43f3-bde3-44f90aadb48c.png)